components/TemplateListItem/TemplateListItem.module.scss

```diff
@@ -0,0 +1,110 @@
+@import '../../styles/responsive.scss';
```

Comment:
L1: I think you need to write the import like `@import '@/styles/responsive.scss';`, because I was seeing different webpack errors, like this one: `<w> while serializing webpack/lib/cache/PackFileCacheStrategy.PackContentItems -> webpack/lib/NormalModule -> Array { 1 items } -> webpack/lib/ModuleWarning -> Warning
<w> [webpack.cache.PackFileCacheStrategy] Skipped not serializable cache item 'Compilation/modules|javascript/auto|/Users/jshin/dev/dmsp_frontend_prototype/node_modules/next/dist/build/webpack/loaders/next-flight-css-loader.js??ruleSet[1].rules[14].oneOf[9].use[0]!/Users/jshin/dev/dmsp_frontend_prototype/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[14].oneOf[9].use[1]!/Users/jshin/dev/dmsp_frontend_prototype/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[14].oneOf[9].use[2]!/Users/jshin/dev/dmsp_frontend_prototype/node_modules/next/dist/build/webpack/loaders/resolve-url-loader/index.js??ruleSet[1].rules[14].oneOf[9].use[3]!/Users/jshin/dev/dmsp_frontend_prototype/node_modules/next/dist/compiled/sass-loader/cjs.js??ruleSet[1].rules[14].oneOf[9].use[4]!/Users/jshin/dev/dmsp_frontend_prototype/components/TemplateListItem/TemplateListItem.module.scss|ssr': No serializer registered for Warning` showing up as I was navigating on the app.

Reply:
Good spot